### DISCUSSION TITLE
Problem with entry point of libopenal

### DISCUSSION THREAD

**Discussion by Aques** [bastianvandenbout/LISEM#8] [Q&A]
Hello, I Successfully compiled the LISEM program with the steps given in [Lisem Compiling](https://lisemmodel.com/docs/compiling/) and the answer given by [bastianvandenbout](https://github.com/bastianvandenbout) on the question [Problem with CMake Configuration](https://github.com/bastianvandenbout/LISEM/discussions/6).

I made a QT kit in the QTcreator given in mysys2 as given in [Set up QTcreator with mysys2](https://gist.github.com/Bluexin/1c58d9b6d707f755b66a94314e3dfd32) discussion.

The binary created was placed in external/runtime
All the missing dll were pasted from external/lib to external/runtime but I am facing following issue,
![image](https://user-images.githubusercontent.com/36673324/193743075-7b157dd3-a3da-4d2c-a282-f8e6b691fac2.png)

Kindly Help,
Thank You

**Comment by bastianvandenbout** [comment]
Hi,

This might be an issue with the runtime dll collection.
Does the compiled executable work when placed in the folder from the 0.2.4 web download?
This download should contain the correct run-time dll's.

If not, it might be good to check which dll's that are linked against do not match in size/creation time with the dlls in the runtime directory. Typically this error indicates a mismatch in my experience.
Hope this helps, let me know!

Kind regards,
bastian

**Reply by Aques** [reply]
Hi,

Yes it was problem of wrong version of dll. I compiled using QTCreator in mysys2 after installing using pacman so the dlls were updated there and some dlls in runtime folder were old. So I replaced them with the dlls using in mysys2.

Thanks for the reply.

Kind regards,
Aques
